### ISSUE TITLE
Make Exposed `_log` Method Public?

### ISSUE THREAD

**Issue by CodeMan99** [sindresorhus/gulp-debug#42]
Would you consider explicitly exposing [_log](https://github.com/sindresorhus/gulp-debug/blob/8108fdfafd0f6b010071439b064668871ec3a6ec/index.js#L50-L51)?

I am using [`gulp-debug`](https://github.com/sindresorhus/gulp-debug) to inspect a [`vinyl-fs`](https://github.com/gulpjs/vinyl-fs) pipeline in a server. In this case I would rather the output be printed by [`debug`](https://github.com/visionmedia/debug) instead of [`fancy-log`](https://github.com/js-cli/fancy-log).

Here is what I am currently doing.

```javascript
var debug = require('debug')('ciq-license-server:pipeline');
var gulpDebug = require('gulp-debug');
var stream = require('stream');

if (gulpDebug._log) {
	gulpDebug._log = function() {
		debug(...arguments);
	};
} else {
	throw new Error('gulp-debug is not exposing "_log" function');
}

module.exports = function(options) {
	return debug.enabled ? gulpDebug(options) : new stream.PassThrough({objectMode: true});
};
```

**Comment by sindresorhus** [comment]
Yeah, sure. PR welcome for a `logger` option. Should come with docs and a test. :)

**Comment by sindresorhus** [comment]
Fixed by #43
